src/ChainRulesTestUtils.jl

```diff
@@ -12,7 +12,8 @@ using Test
 const _fdm = central_fdm(5, 1; max_range=1e-2)
 
 export TestIterator
-export check_equal, test_scalar, frule_test, rrule_test, generate_well_conditioned_matrix
+export check_equal, test_scalar, test_frule, test_rrule, generate_well_conditioned_matrix
+export Auto, ⊢
```

Comment:
Don't need to export `Auto()` as I made the output tangent into a kwarg, so user doesn't need to specify to do it Auto, they just need to not pass that kwarg

Reply:
```suggestion
export ⊢
```